### PR TITLE
Add video tutorials and tag for video tutorials

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
Added a videos tag to the tutorials section for video tutorials

## Description

Added the following videos:

FreeCodeCamp End to end: https://www.youtube.com/watch?v=M576WGiDBdQ
NFTs & Hardhat: https://www.youtube.com/watch?v=9oERTH9Bkw0
Leveraged Trading & Defi: https://www.youtube.com/watch?v=TmNGAvI-RUA
Smart Contract Upgrades: https://www.youtube.com/watch?v=bdXJmWajZRY
Flash Loans: https://www.youtube.com/watch?v=Aw7yvGFtOvI
NFTs & Brownie: https://www.youtube.com/watch?v=p36tXHX1JD8
ERC20s: https://www.youtube.com/watch?v=8rpir_ZSK1g

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/3868

### CODE REVIEW

**Comment by PatrickAlphaC** [comment]
Let me know what you think!

A lot of these tutorials also have written abridged versions if you'd like those instead, so we can have both the written and video, and the users can pick. 

But in any case, looking to see what you think is best for the community.

File: src/data/tutorials.json [hunk 1]
```diff
@@ -1,4 +1,88 @@
...
+    "title": "Solidity, Blockchain, and Smart Contract Course – Beginner to Expert Python Tutorial",
+    "description": "This course will give you a full introduction into all of the core concepts in blockchain, smart contracts, solidity, NFTs/ERC721s, ERC20s, Coding Decentralized Finance (DeFi), python and solidity, Chainlink, Ethereum, upgradable smart contracts, and full stack blockchain development. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "brownie", "web3.py", "smart-contracts", "react", "blockchain", "video", "IPFS", "oracles", "NFT", "ERC20", "upgrades", "python", "mocking", "testing", "Aave", "chainlink", "ganache"],
+    "skillLevel": "beginner",
+    "timeToRead": "960",
+    "lang": "en",
+    "publishDate": "09/09/2021"
+  },
```
Comment (minimalsm) [inline src/data/tutorials.json:13]:
```suggestion
  {
    "url": "https://www.youtube.com/watch?v=M576WGiDBdQ",
    "title": "Solidity, Blockchain, and Smart Contract Course",
    "description": "This course will give you a full introduction into all of the core concepts in blockchain, smart contracts, solidity, NFTs/ERC721s, ERC20s, Coding Decentralized Finance (DeFi), python and solidity, Chainlink, Ethereum, upgradable smart contracts, and full stack blockchain development. ",
    "author": "Patrick Collins",
    "authorGithub": "https://github.com/PatrickAlphaC",
    "tags": ["solidity", "brownie", "web3.py", "smart-contracts", "react", "blockchain", "video", "IPFS", "oracles", "NFT", "ERC20", "upgrades", "python", "mocking", "testing", "Aave", "chainlink", "ganache"],
    "skillLevel": "beginner",
    "timeToRead": "960",
    "lang": "en",
    "publishDate": "09/09/2021"
  },
```

File: src/data/tutorials.json [hunk 2]
```diff
@@ -1,4 +1,88 @@
...
+    "title": "How to make NFT Art with On-Chain Metadata | FULL HARDHART / JS TUTORIAL!",
+    "description": "We explore the world of using SVGs to generate random NFT ImageURIs and Metadata 100% on-chain.  ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "ethers.js", "hardhat", "NFT", "smart-contracts", "javascript", "chainlink", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "180",
+    "lang": "en",
+    "publishDate": "09/03/2021"
+  },
```
Comment (minimalsm) [inline src/data/tutorials.json:25]:
```suggestion
  {
    "url": "https://www.youtube.com/watch?v=9oERTH9Bkw0",
    "title": "How to make NFT Art with On-Chain Metadata",
    "description": "We explore the world of using SVGs to generate random NFT ImageURIs and Metadata 100% on-chain.  ",
    "author": "Patrick Collins",
    "authorGithub": "https://github.com/PatrickAlphaC",
    "tags": ["solidity", "ethers.js", "hardhat", "NFT", "smart-contracts", "javascript", "chainlink", "video"],
    "skillLevel": "beginner",
    "timeToRead": "180",
    "lang": "en",
    "publishDate": "09/03/2021"
  },
```

File: src/data/tutorials.json [hunk 3]
```diff
@@ -1,4 +1,88 @@
...
+    "title": "Leveraged Trading in DeFi | Python, Brownie, Aave, Uni/SushiSwap",
+    "description": "Leveraged trading is a common strategy to use in traditional finance, and they are even easier to do in DeFi. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "34",
+    "lang": "en",
+    "publishDate": "06/29/2021"
+  },
```
Comment (minimalsm) [inline src/data/tutorials.json:37]:
```suggestion
  {
    "url": "https://www.youtube.com/watch?v=TmNGAvI-RUA",
    "title": "Leveraged Trading in DeFi,
    "description": "Leveraged trading is a common strategy to use in traditional finance, and they are even easier to do in DeFi. ",
    "author": "Patrick Collins",
    "authorGithub": "https://github.com/PatrickAlphaC",
    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video"],
    "skillLevel": "beginner",
    "timeToRead": "34",
    "lang": "en",
    "publishDate": "06/29/2021"
  },
```

File: src/data/tutorials.json [hunk 4]
```diff
@@ -1,4 +1,88 @@
...
+    "tags": ["solidity", "ethers.js", "hardhat", "NFT", "smart-contracts", "javascript", "chainlink", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "180",
+    "lang": "en",
+    "publishDate": "09/03/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=TmNGAvI-RUA",
+    "title": "Leveraged Trading in DeFi | Python, Brownie, Aave, Uni/SushiSwap",
+    "description": "Leveraged trading is a common strategy to use in traditional finance, and they are even easier to do in DeFi. ",
```
Comment (minimalsm) [inline src/data/tutorials.json:29]: > Leveraged trading is a common strategy to use in traditional finance, and they are even easier to do in DeFi.

This description doesn't make sense 😅 maybe we could reword to: `and leveraged trades are even easier to do in DeFi.`

File: src/data/tutorials.json [hunk 5]
```diff
@@ -1,4 +1,88 @@
...
+    "title": "Upgrading your Smart Contracts | A Tutorial & Introduction",
+    "description": "Learn how to make contracts that use flash loans. Using Brownie, Solidity, Aave. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video", "defi"],
+    "skillLevel": "intermediate",
+    "timeToRead": "17",
+    "lang": "en",
+    "publishDate": "04/25/2021"
+  },
```
Comment (minimalsm) [inline src/data/tutorials.json:49]:
```suggestion
  {
    "url": "https://www.youtube.com/watch?v=bdXJmWajZRY",
    "title": "Using flash loans in your smart contracts",
    "description": "Learn how to make contracts that use flash loans. Using Brownie, Solidity, Aave. ",
    "author": "Patrick Collins",
    "authorGithub": "https://github.com/PatrickAlphaC",
    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video", "defi"],
    "skillLevel": "intermediate",
    "timeToRead": "17",
    "lang": "en",
    "publishDate": "04/25/2021"
  },
```

File: src/data/tutorials.json [hunk 6]
```diff
@@ -1,4 +1,88 @@
...
+    "url": "https://www.youtube.com/watch?v=8rpir_ZSK1g",
+    "title": "Create your own Blockchain ERC20 Token | Python, Brownie, Solidity",
+    "description": "Deploy your smart contract to Opensea, end-to-end. The Ultimate/end-to-end tutorial.",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "smart-contracts", "python", "openzeppelin", "erc20", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "30",
+    "lang": "en",
+    "publishDate": "05/24/2021"
```
Comment (minimalsm) [inline src/data/tutorials.json:84]:
```suggestion
    "url": "https://www.youtube.com/watch?v=8rpir_ZSK1g",
    "title": "Create your own Blockchain ERC20 Token",
    "description": "Deploy your smart contract to Opensea, end-to-end. The Ultimate/end-to-end tutorial.",
    "author": "Patrick Collins",
    "authorGithub": "https://github.com/PatrickAlphaC",
    "tags": ["solidity", "web3.py", "brownie", "smart-contracts", "python", "openzeppelin", "erc20", "video"],
    "skillLevel": "beginner",
    "timeToRead": "30",
    "lang": "en",
    "publishDate": "05/24/2021"
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @PatrickAlphaC thanks so much for this. Loads of great content here for the tutorials section and it definitely gives us a good push to improve our tutorial's section to utilise video content in the future.

I've 'de-YouTubed' this a bit by suggesting we remove tags in the titles as they are all already covered in the tags section.

Could you also have a quick scan through the titles and descriptions for me? I think some of them got mixed up in your PR.

Screenshot 2021-09-16 at 14 42 33

File: src/data/tutorials.json (hunk 5 above)
Comment (PatrickAlphaC) [reply src/data/tutorials.json:49]: I think this one is correct no? Does that link give you a flash loan video? @minimalsm

File: src/data/tutorials.json (hunk 4 above)
Comment (PatrickAlphaC) [reply src/data/tutorials.json:29]: Yeaaaahhh my solidity is better than my english.

**Comment by PatrickAlphaC** [comment]
Thanks for the comments @minimalsm! I've updated accordingly. 

I think the links actually all look in the correct spot to me no? Do you see one that is mixed up?

File: src/data/tutorials.json [hunk 7]
```diff
@@ -1,4 +1,88 @@
 [
+  {
+    "url": "https://www.youtube.com/watch?v=M576WGiDBdQ",
+    "title": "Solidity, Blockchain, and Smart Contract Course",
+    "description": "This course will give you a full introduction into all of the core concepts in blockchain, smart contracts, solidity, NFTs/ERC721s, ERC20s, Coding Decentralized Finance (DeFi), python and solidity, Chainlink, Ethereum, upgradable smart contracts, and full stack blockchain development. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "brownie", "web3.py", "smart-contracts", "react", "blockchain", "video", "IPFS", "oracles", "NFT", "ERC20", "upgrades", "python", "mocking", "testing", "Aave", "chainlink", "ganache"],
```
Comment (minimalsm) [inline src/data/tutorials.json:8]:
```suggestion
    "tags": ["solidity", "brownie", "web3.py", "smart contracts", "react", "blockchain", "video", "IPFS", "oracles", "NFT", "ERC-20", "upgrades", "python", "mocking", "testing", "Aave", "chainlink", "ganache"],
```

File: src/data/tutorials.json [hunk 8]
```diff
@@ -1,4 +1,88 @@
...
+    "lang": "en",
+    "publishDate": "09/09/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=9oERTH9Bkw0",
+    "title": "How to make NFT Art with On-Chain Metadata",
+    "description": "We explore the world of using SVGs to generate random NFT ImageURIs and Metadata 100% on-chain.  ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "ethers.js", "hardhat", "NFT", "smart-contracts", "javascript", "chainlink", "video"],
```
Comment (minimalsm) [inline src/data/tutorials.json:20]:
```suggestion
    "tags": ["solidity", "ethers.js", "hardhat", "NFT", "smart contracts", "javascript", "chainlink", "video"],
```

File: src/data/tutorials.json [hunk 9]
```diff
@@ -1,4 +1,88 @@
...
+    "lang": "en",
+    "publishDate": "09/03/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=TmNGAvI-RUA",
+    "title": "Leveraged Trading in DeFi",
+    "description": "Leveraged trading is a common strategy in traditional finance, and leveraged trades are even easier to do in DeFi",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video"],
```
Comment (minimalsm) [inline src/data/tutorials.json:32]:
```suggestion
    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart contracts", "python", "video"],
```

File: src/data/tutorials.json [hunk 10]
```diff
@@ -1,4 +1,88 @@
...
+    "lang": "en",
+    "publishDate": "06/29/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=bdXJmWajZRY",
+    "title": "Upgrading your Smart Contracts | A Tutorial & Introduction",
+    "description": "Learn how to make contracts that use flash loans. Using Brownie, Solidity, Aave. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video", "defi"],
```
Comment (minimalsm) [inline src/data/tutorials.json:44]:
```suggestion
    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart contracts", "python", "video", "defi"],
```

File: src/data/tutorials.json [hunk 11]
```diff
@@ -1,4 +1,88 @@
...
+    "lang": "en",
+    "publishDate": "04/25/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=Aw7yvGFtOvI",
+    "title": "Aave Flash Loan Tutorial",
+    "description": "All about upgradable smart contracts, proxies, and using delegatecall in your solidity. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "upgrades", "smart-contracts", "python", "openzeppelin", "proxies", "video"],
```
Comment (minimalsm) [inline src/data/tutorials.json:56]:
```suggestion
    "tags": ["solidity", "web3.py", "brownie", "upgrades", "smart contracts", "python", "openzeppelin", "proxies", "video"],
```

File: src/data/tutorials.json [hunk 12]
```diff
@@ -1,4 +1,88 @@
...
+    "lang": "en",
+    "publishDate": "05/24/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=p36tXHX1JD8",
+    "title": "NFT/ERC-721/Collectible END-TO-END TUTORIAL | Deploy, List on Opensea, Host Metadata on IPFS",
+    "description": "Build your own ERC20 token using Brownie, Python, and Solidity. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "upgrades", "smart-contracts", "python", "openzeppelin", "chainlink", "video"],
```
Comment (minimalsm) [inline src/data/tutorials.json:68]:
```suggestion
    "tags": ["solidity", "web3.py", "brownie", "upgrades", "smart contracts", "python", "openzeppelin", "chainlink", "video"],
```

File: src/data/tutorials.json [hunk 13]
```diff
@@ -1,4 +1,88 @@
...
+    "lang": "en",
+    "publishDate": "05/09/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=8rpir_ZSK1g",
+    "title": "Create your own Blockchain ERC20 Token",
+    "description": "Deploy your smart contract to Opensea, end-to-end. The Ultimate/end-to-end tutorial.",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "smart-contracts", "python", "openzeppelin", "erc20", "video"],
```
Comment (minimalsm) [inline src/data/tutorials.json:80]:
```suggestion
    "tags": ["solidity", "web3.py", "brownie", "smart contracts", "python", "openzeppelin", "ERC-20", "video"],
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Just some changes to keep the tags consistent :)

**Comment by minimalsm** [comment]
@PatrickAlphaC thanks for working on this and supplying great content generally - just waiting on the deploy preview to build so I can double-check this before putting it out 👍

**Comment by PatrickAlphaC** [comment]
Boom!! Awesome!

File: src/data/tutorials.json [hunk 14]
```diff
@@ -1,4 +1,88 @@
...
+    "tags": ["solidity", "brownie", "web3.py", "smart contracts", "react", "blockchain", "video", "IPFS", "oracles", "NFT", "ERC-20", "upgrades", "python", "mocking", "testing", "Aave", "chainlink", "ganache"],
+    "skillLevel": "beginner",
+    "timeToRead": "960",
+    "lang": "en",
+    "publishDate": "09/09/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=9oERTH9Bkw0",
+    "title": "How to make NFT Art with On-Chain Metadata",
+    "description": "We explore the world of using SVGs to generate random NFT ImageURIs and Metadata 100% on-chain.  ",
```
Comment (minimalsm) [inline src/data/tutorials.json:17]:
```suggestion
    "description": "Explore the world of using SVGs to generate random NFT ImageURIs and Metadata 100% on-chain.",
```

File: src/data/tutorials.json [hunk 15]
```diff
@@ -1,4 +1,88 @@
...
+    "tags": ["solidity", "web3.py", "brownie", "upgrades", "smart contracts", "python", "openzeppelin", "chainlink", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "17",
+    "lang": "en",
+    "publishDate": "05/09/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=8rpir_ZSK1g",
+    "title": "Create your own Blockchain ERC20 Token",
+    "description": "Deploy your smart contract to Opensea, end-to-end. The Ultimate/end-to-end tutorial.",
```
Comment (minimalsm) [inline src/data/tutorials.json:77]:
```suggestion
    "description": "Deploy your smart contract to Opensea, end-to-end.",
```

Comment (minimalsm) [reply src/data/tutorials.json:77]: Missed this one

File: src/data/tutorials.json [hunk 16]
```diff
@@ -1,4 +1,88 @@
...
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart contracts", "python", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "34",
+    "lang": "en",
+    "publishDate": "06/29/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=bdXJmWajZRY",
+    "title": "Upgrading your Smart Contracts | A Tutorial & Introduction",
+    "description": "Learn how to make contracts that use flash loans. Using Brownie, Solidity, Aave. ",
```
Comment (minimalsm) [inline src/data/tutorials.json:41]:
```suggestion
    "description": "Learn how to make contracts that use flash loans. Using Brownie, Solidity, Aave.",
```

File: src/data/tutorials.json [hunk 17]
```diff
@@ -1,4 +1,88 @@
...
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart contracts", "python", "video", "defi"],
+    "skillLevel": "intermediate",
+    "timeToRead": "17",
+    "lang": "en",
+    "publishDate": "04/25/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=Aw7yvGFtOvI",
+    "title": "Aave Flash Loan Tutorial",
+    "description": "All about upgradable smart contracts, proxies, and using delegatecall in your solidity. ",
```
Comment (minimalsm) [inline src/data/tutorials.json:53]:
```suggestion
    "description": "All about upgradable smart contracts, proxies, and using delegatecall in your solidity.",
```

File: src/data/tutorials.json [hunk 18]
```diff
@@ -1,4 +1,88 @@
...
+    "tags": ["solidity", "web3.py", "brownie", "upgrades", "smart contracts", "python", "openzeppelin", "proxies", "video"],
+    "skillLevel": "intermediate",
+    "timeToRead": "30",
+    "lang": "en",
+    "publishDate": "05/24/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=p36tXHX1JD8",
+    "title": "NFT/ERC-721/Collectible END-TO-END TUTORIAL | Deploy, List on Opensea, Host Metadata on IPFS",
+    "description": "Build your own ERC20 token using Brownie, Python, and Solidity. ",
```
Comment (minimalsm) [inline src/data/tutorials.json:65]:
```suggestion
    "description": "Build your own ERC20 token using Brownie, Python, and Solidity.",
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

**Comment by minimalsm** [comment]
Awesome! Thanks for this @PatrickAlphaC! Should be live soon 😀

**Comment by PatrickAlphaC** [comment]
!!!!!

Amazing! Woot woot!!

**Pull request merged**